content/docs/user-guide/networking/multiplayer/spawning.md

```diff
@@ -27,8 +29,15 @@ The `OnPlayerLeave` method of `IMultiplayerSpawner` provides a hook so that when
 
 `OnPlayerLeave` also takes a disconnect reason which allows responding to different kinds of disconnects. For example, it may be undesirable to clean up objects if a player times out if they can attempt to reconnect to the session.
 
-### Spawning examples in MultiplayerSample
 
-A practical example of an implementation for a spawner is [`MultiplayerSampleSystemComponent`](https://github.com/o3de/o3de-multiplayersample/blob/development/Gem/Code/Source/MultiplayerSampleSystemComponent.cpp) in the  [MultiplayerSample project](https://github.com/o3de/o3de-multiplayersample/). MultiplayerSample implements a "round robin"-style spawning system that gathers entities with `NetworkPlayerSpawnerComponents`. `MultiplayerSampleSystemComponent` then queries that system during `OnPlayerJoin`. `OnPlayerLeave` simply marks the entity passed in for removal. 
+## Examples in MultiplayerSample Project
```

Comment:
```suggestion
## Examples in MultiplayerSample project
```
Consider changing to [sentence case for H2 headers](https://www.o3de.org/docs/contributing/to-docs/style-guide/format/#topic-headings). This is consistent with the capitalization (lowercase "project") in the first sentence of the following paragraph ("...in the MultiplayerSample project"). If you prefer to match the capitalization of the `README.md` title ("MultiplayerSample Project"), then change the first sentence occurrence to match, for consistency.

Reply:
I think "Project" should remain capitalized here because it's the proper name - MultiplayerSample Project. (See [repo](https://github.com/o3de/o3de-multiplayersample)). 
@paucom Thoughts?

Reply:
@chanmosq That change is fine to me. However, as I mentioned in my previous comment ("f you prefer to match the capitalization of the `README.md` title...."), in this case, I recommend a corresponding change in the first sentence of the following paragraph. I'll add a comment for that change instead.